libraries/AP_HAL_SITL/SITL_State.cpp

```diff
@@ -469,7 +469,12 @@ void SITL_State::_simulator_servos(struct sitl_input &input)
                 }
             } else {
                 // simulate simple battery setup
-                float throttle = motors_on?(input.servos[2]-1000) / 1000.0f:0;
+                float throttle;
+                if (_vehicle == APMrover2) {
```

Comment:
Isn't this also relevant for sub?

And Planes that support reverse throttle? (which I don't think we have a simulator for...)

Still, not after perfection.